### DISCUSSION TITLE
Paste multi-paragraph text as node's children

### DISCUSSION THREAD

**Discussion by antanasb** [nadrad/h-m-m#78] [Ideas]
That could be usefull importing feature: populate node's children from the multi-paragraph text in the clipboard (one paragraph per child node).

**Comment by nadrad** [comment]
So, when you paste a multi-paragraph text, each of its paragraphs become a new child for the active node. This is what we already have; are you taking about something else?

**Reply by antanasb** [reply]
OK, thanks, it works! I even didn't have the courage to try it out. It's not documented or I missed somthing?

**Comment by nadrad** [comment]
This is simply the way that makes more sense in a mind mapping application, and it seems so natural to me that I can't even think about documenting it :)

It also tries to convert things that look like lists into nodes and removes their list symbols. It even identifies some of the hierarchies and creates multiple layers for them when pasting.
